src/System.ComponentModel.Primitives/src/System/ComponentModel/Component.cs

```diff
@@ -0,0 +1,276 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.ComponentModel
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <summary>
+    ///    <para>Provides the default implementation for the 
+    ///    <see cref='System.ComponentModel.IComponent'/>
+    ///    interface and enables object-sharing between applications.</para>
+    /// </summary>
+    [
+        ComVisible(true),
+        //ClassInterface(ClassInterfaceType.AutoDispatch),
+        DesignerCategory("Component")
+    ]
+    public class Component : /*TODO NETSTANDARD2.0 MarshalByRefObject, */IComponent 
+    {
+        /// <summary>
+        ///    <para>Static hask key for the Disposed event. This field is read-only.</para>
```

Comment:
They could -- it looks like we were not consistent across this repo ... so we have other places with these tags. 

I will capture these in a code-clean-up issue.